wremnants/syst_tools.py

```diff
@@ -361,6 +361,17 @@ def add_pdf_hists(results, df, dataset, axes, cols, pdfs, base_name="nominal", a
         else:
             pdfHist = df.HistoBoost(pdfHistName, axes, [*cols, tensorName], tensor_axes=[pdf_ax], storage=hist.storage.Double())
             alphaSHist = df.HistoBoost(alphaSHistName, axes, [*cols, tensorASName], tensor_axes=[as_ax], storage=hist.storage.Double())
+
+            if propagateToHelicity:
+                df=df.Define("unity","1.")
```

Comment:
DefinePerSample is better to define a constant (I think there used to be a definition of unity and zero somewhere in the code in the past, but since the code is not failing CI I assume it was removed, so there is no duplication).

Reply:
Is this really the first time this gets defined? I thought similar was done in other places.

Reply:
Yah probably with a slightly different name.